### ISSUE TITLE
Fixing the inconsistent rendering of haddocks across hackage

### ISSUE THREAD

**Issue by MangoIV** [haskell/hackage-server#1349]
I think I have a pretty "simple" idea for a fix here. How about we
- remove all css from doc tarballs
- ship our own css that adheres with the rest of the style of hackage

That way we wouldn't get outdated styles.

**Comment by MangoIV** [comment]
So afaiu at the moment, how it works is that hackage-server just loads the tarball and then serves whatever file from the tarball that was requested. I guess to replace the css then, whenever we match on a css path, we would serve our own css :3. 

(https://github.com/haskell/hackage-server/blob/962b6d77c54e4cb2491c7be3fe84797c60471782/src/Distribution/Server/Features/Documentation.hs#L296-L298)

**Comment by MangoIV** [comment]
In my opinion user generated archives shouldn't be a thing at all. I think it would be even be a goo good idea to throw away everything that isn't enclosed in the html <body> type and then some tags that may be dangerous.

**Comment by peterbecich** [comment]
Good idea, duplicate to https://github.com/haskell/hackage-server/issues/1329

**Comment by peterbecich** [comment]
For the simplest solution, cannot all these tarballs just be re-generated with the latest CSS?  This would not require an architectural change.  Perhaps an "invisible" version published for every package.
